docroot/modules/humsci/hs_admin/hs_admin.module

```diff
@@ -8,6 +8,23 @@
 /**
  * Implements hook_form_alter().
  */
-function hs_admin_preprocess_node_edit_form(&$variables) {
-  $variables['#attached']['library'][] = 'hs_admin/hs_admin';
+function hs_admin_page_attachments(&$variables) {
+  $logged_in = \Drupal::currentUser()->isAuthenticated();
+
+  if ($logged_in) {
+    $variables['#attached']['library'][] = 'hs_admin/hs_admin';
+  }
+}
+
```

Comment:
Why I changed this: the `node_edit_form` isn't available on some of the required pages, so I made it available on every admin page by checking for authenticated users. I'm personally not worried about perf implications for a logged in user.

Reply:
Good call 👍